static/css/sass/cdr-ui.scss

```diff
@@ -664,8 +673,8 @@ p.spacing {
 }
 
 .item_container_count {
-  background-color: $unc-blue;
-  border: 1px solid $unc-blue;
+  background-color: #1A698C;
```

Comment:
Why not use a variable for these anymore?

Reply:
$unc-blue still gets used in other places. This is a different, darker color, though it does get used a few times. I can add it as a variable